src/misc/lv_text.c

```diff
@@ -649,6 +649,9 @@ static uint32_t lv_text_utf8_next(const char * txt, uint32_t * i)
     uint32_t i_tmp = 0;
     if(i == NULL) i = &i_tmp;
 
+    /* Ensure the string is not null */
+    if(txt == NULL || txt[*i] == '\0') return result;
```

Comment:
Suggest to track potential NULL strings by adding a LOG_WRN or a LOG_ERR

Reply:
Added a warning logging :^)